lms/djangoapps/teams/static/teams/templates/team-profile.underscore

```diff
@@ -4,8 +4,10 @@
                 data-read-only="<%- readOnly %>"
                 data-user-create-comment="<%- !readOnly %>"
                 data-user-create-subcomment="<%- !readOnly %>">
-            <% if (!readOnly) { %>
-                <button type="button" class="btn new-post-btn"><span class="icon fa fa-edit new-post-icon" aria-hidden="true"></span><%- gettext("New Post") %></button>
+            <% if ( !readOnly) { %>
+                <button type="button" class="btn-neutral btn-small new-post-btn">
+                    <i class="icon fa fa-edit" aria-hidden="true"></i> <%- gettext("New Post") %>
```

Comment:
Good catch! I must have messed up the rebase. I've removed the icon now and changed the label to match all the other "Add a Post" buttons (what was "New Post").